src/components/OcDrop.vue

```diff
@@ -26,29 +28,22 @@ export default {
       required: false,
       default: () => uniqueId("oc-drop-"),
     },
-    /**
-     * CSS selector of the element to maintain the drop's visibility. see https://getuikit.com/docs/drop#boundary
-     **/
-    boundary: {
-      type: [String, Boolean],
-      default: false,
-    },
     /**
      * CSS selector for the element to be used as toggle. By default, the preceding element is used.
      **/
     toggle: {
       type: String,
-      default: "- *",
+      default: "",
     },
     /**
      * The position of the drop.
      **/
     position: {
       type: String,
-      default: "bottom-left",
+      default: "bottom-start",
       validator: value => {
         return value.match(
-          /((top|bottom)-(left|center|right|justify)|(left|right)-(top|center|bottom))/
+          /((top|right|bottom|left|auto)|(top|right|bottom|left|auto)-(start|end))/
```

Comment:
Could you add a line to the docblock of this prop about the valid combinations? Like `Use top, right, bottom, left or auto for centered placement or add '-start'  or '-end'  as a suffix for altering the alignment`?
Also: is `auto-start` really a valid combination? 🤔

Reply:
Yea it’s valid, but to be honest I haven’t tried the auto prop...

 // default
  placement: 'top',

  // full list:
  placement: 'top-start',
  placement: 'top-end',

  placement: 'right',
  placement: 'right-start',
  placement: 'right-end',

  placement: 'bottom',
  placement: 'bottom-start',
  placement: 'bottom-end',

  placement: 'left',
  placement: 'left-start',
  placement: 'left-end',

  // choose the side with most space
  placement: 'auto',
  placement: 'auto-start',
  placement: 'auto-end',

Reply:
Please don't add the full list. 😁 After all it's devdocs, so people reading it will be capable of understanding a hint about the combinations. Just the regex would be nice to avoid in the docs. ;-)